FoodplannerServices/Image/FoodImageService.cs

```diff
@@ -8,6 +9,8 @@ namespace FoodplannerServices.Image;
 
 public class FoodImageService(IImageService imageService, IFoodImageRepository foodImageRepository) : IFoodImageService
```

Comment:
The mapper needs to be dependency injected in, i do not like using these constructors but is should be consistent so either
```suggestion
public class FoodImageService(IImageService imageService, IFoodImageRepository foodImageRepository, IMapper mapper) : IFoodImageService
```
and change all uses of the mapper from _mapper to mapper. Preferably move all dependency injected classes to their own private readonly fields and assign them in a constructor.